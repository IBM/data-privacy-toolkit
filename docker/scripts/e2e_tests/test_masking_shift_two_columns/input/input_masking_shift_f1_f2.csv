M,22,1001
F,23,1002
M,27,1003
