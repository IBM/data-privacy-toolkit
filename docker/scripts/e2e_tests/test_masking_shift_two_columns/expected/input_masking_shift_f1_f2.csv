M,1022.000000,901.000000
F,1023.000000,902.000000
M,1027.000000,903.000000
